lib/h_pyramid_sentry/h_pyramid_sentry/__init__.py

```diff
@@ -22,14 +21,28 @@ def report_exception(exc=None):
 
 def includeme(config):
     """Set up the error tracking service."""
+    filters = [
+        # Allow functions to be passed as strings: e.g. "module.function"
+        config.maybe_dotted(_filter)
+        for _filter in config.registry.settings.get("h_pyramid_sentry.filters", [])
+    ]
+
+    if config.registry.settings.get("h_pyramid_sentry.retry_support"):
```

Comment:
Similar to the dottedname thing for turning a string into a function, you can use Pyramid's [`asbool`](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/settings.html#pyramid.settings.asbool) here which will turn strings like `"true"` etc into bools (but not touch it if the value is already a bool). (This is because the app could set `h_pyramid_sentry.retry_support` in the config file rather than in Python.)

Also see https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html#adding-a-custom-setting

Reply:
Fixed in the new repo